smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ClientRuntimeTypes.kt

```diff
@@ -97,6 +97,12 @@ object ClientRuntimeTypes {
         val PaginateToken = runtimeSymbol("PaginateToken")
         val PaginatorSequence = runtimeSymbol("PaginatorSequence")
     }
+
+    object Test {
+        val CrtXCBaseTestCase = buildSymbol {
+            name = "CrtXCBaseTestCase"
```

Comment:
What is the purpose of this build symbol (and the one for `XCTestCase`)?  Is this to facilitate code-generating test cases?

Reply:
Yes, symbol for generating test base class. Given this is client runtime symbol. We need this in this repo.